app/assets/javascripts/account/AccountController.js.coffee

```diff
@@ -89,12 +90,11 @@ AccountController = ($scope, $state, $document, $translate, AccountService, Shor
       $scope.handleErrorState()
 
   $scope.updateEmail = ->
-    # form = $scope.form.accountEmail
     $scope.form.current = $scope.form.accountEmail
-    AccountService.updateAccount()
+    # AccountService.updateAccount()
+    false
```

Comment:
@kaptron is there a reason why you returned false here -- was that just something temp for the feature? Can i revert it back? Might be good to go through some of this with you before I deliver the story